llvm/test/CodeGen/AMDGPU/agpr-register-count.ll

```diff
@@ -157,10 +157,8 @@ declare void @undef_func()
 ; GFX908: .amdhsa_next_free_vgpr 32
 ; GFX90A: .amdhsa_next_free_vgpr 64
 ; GFX90A: .amdhsa_accum_offset 32
-; GCN908: NumVgprs: 128
-; GCN908: NumAgprs: 128
-; GCN90A: NumVgprs: 256
-; GCN90A: NumAgprs: 256
+; GCN:    NumVgprs: 32
+; GCN:    NumAgprs: 32
```

Comment:
This should have been part of https://reviews.llvm.org/D117504 "AMDGPU: Convert AMDGPUResourceUsageAnalysis to a Module pass"